src/examples/install-kubeconfig.yml

```diff
@@ -0,0 +1,18 @@
+description: |
+  Install kubeconfig
+
+usage:
+  version: 2.1
+
+  orbs:
+    kube-orb: circleci/kubernetes@1.0.0
+
+  jobs:
+    build:
+      macos:
+        xcode: "9.0"
+      steps:
+        - checkout
+
+        - kube-orb/install-kubeconfig:
```

Comment:
Could you update this example since the kubeconfig parameter is now an environment variable name?